crates/telio-traversal/src/endpoint_providers/upnp/mod.rs

```diff
@@ -21,49 +24,80 @@ use telio_utils::{
     telio_log_debug, telio_log_info, PinnedSleep,
 };
 use telio_wg::{DynamicWg, WireGuard};
-use tokio::{net::UdpSocket, sync::Mutex};
+use tokio::{net::UdpSocket, sync::Mutex, time::sleep};
 
 #[cfg(test)]
 use mockall::automock;
 #[cfg(test)]
-const UPNP_INTERVAL: Duration = Duration::from_millis(100);
+const UPNP_INTERVAL_MS: Duration = Duration::from_millis(100);
 
 const MAX_SUPPORTED_PACKET_SIZE: usize = 1500;
 #[cfg(not(test))]
-const UPNP_INTERVAL: Duration = Duration::from_secs(5);
```

Comment:
TODO: Fix initial 5s delay caused by this timeout. (STUN EP is facing the same issue)